proposed/longhorn/latest/README.md

```diff
@@ -9,22 +9,34 @@ Longhorn is 100% open source software. Project source code is spread across a nu
 ## Prerequisites
 
 1. Docker v1.13+
-2. Kubernetes v1.8+
+2. Kubernetes v1.8+ cluster with 1 or more nodes and Mount Propagation feature enabled. If your Kubernetes cluster was provisioned by Rancher v2.0.7+ or later, MountPropagation feature is enabled by default. [Check your Kubernetes environment now](https://github.com/rancher/longhorn#environment-check-script). If MountPropagation is disabled, legacy K8s FlexVolume driver will be deployed instead of the newer K8s CSI driver. Volumes backed by a Base Image will also be disabled.
 3. Make sure `curl`, `findmnt`, `grep`, `awk` and `blkid` has been installed in all nodes of the Kubernetes cluster.
 4. Make sure `open-iscsi` has been installed in all nodes of the Kubernetes cluster. For GKE, recommended Ubuntu as guest OS image since it contains `open-iscsi` already.
 
 ## Uninstall Longhorn
 
-In order to uninstall Longhorn, user need to remove all the volumes first:
-```
-kubectl -n longhorn-system delete lhv --all
-```
+**Do not attempt to immediately delete Longhorn App from Rancher UI. Longhorn propagates device mounts which can damage the host if not unmounted properly.** Follow this procedure instead.
 
-After confirming all the volumes are removed, then Longhorn can be easily uninstalled using:
+1. To prevent damage to the Kubernetes cluster, we recommend deleting all Kubernetes workloads using Longhorn volumes (Deployments, StatefulSets, DaemonSets, etc).
```

Comment:
Need to add a document in the main repo (e.g. `docs/chart.md`). It's better if we just link and copy the document here, rather than let this document live on its own in the chart repo.